src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1 import no_path_templates_pb2
+          >>> api = NoTemplatesServiceApi()
```

Comment:
Nevermind, I figured it out myself. It was inappropriate of me to use PythonImportHandler, which is intended for surface import statements. This is a different use case, and not worth attempting to hijack PythonImportHandler to support it. Instead I'm directly using PythonImport from PythonGapicContext to generate the import statement I want.